openpathsampling/pathmover.py

```diff
@@ -2338,8 +2322,13 @@ def __init__(self, **kwargs):
         super(MoveDetails, self).__init__(**kwargs)
 
 
+# leave this for potential backwards compatibility
 class SampleDetails(Details):
     """Details of a sample
+
+    Notes
+    -----
+    Deprecated. Will not present in future versions
```

Comment:
I was looking through some numpydoc format stuff earlier, and there's a specific way to make deprecation warnings:

```rst
.. note:: Deprecated in NumPy 1.6.0
          `ndobj_old` will be removed in NumPy 2.0.0, it is replaced by
          `ndobj_new` because the latter works also with array subclasses.
```

I'd say we're deprecating it in 0.9.2 and removing in ... 2.0? or even 1.0?